src/Pages/WishList/WishList.scss

```
@@ -0,0 +1,232 @@
+@import "../../Styles/common.scss";
+
+@font-face {
+  font-family: "Neue Haas Grotesk-Bold";
+  src: local("Neue-HaasGrotesk-Text"), url("/public/") format("woff2"),
+    url("http://localhost:3000/Font/NeueHaasGroteskText-Bold.otf")
+      format("opentype");
+}
+
+@font-face {
+  font-family: "Neue Haas Grotesk-Regular";
+  src: local("Neue-HaasGrotesk-Text"), url("/public/") format("woff2"),
+    url("http://localhost:3000/Font/NeueHaasGroteskText-Regular.otf")
+      format("opentype");
+}
+
+@mixin flexRow {
+  display: flex;
+  flex-direction: row;
+}
+
+@mixin flexBtwn {
+  display: flex;
+  justify-content: space-between;
+}
+
+@mixin hassBold {
+  font-family: Neue Haas Grotesk-Bold, Arial, sans-serif;
+  font-size: 13px;
+  line-height: 13px;
+}
+
+@mixin hassBoldSmall {
+  font-family: Neue Haas Grotesk-Bold, Arial, sans-serif;
+  font-size: calc(10rem / 16);
+  line-height: 13px;
+}
+
+@mixin hassRegular {
+  font-family: Neue Haas Grotesk-Regular, Arial, sans-serif;
+  font-size: calc(10rem / 16);
+  line-height: 13px;
+}
+
+@mixin buttonNone {
+  border: 0;
+  background-color: rgba(255, 255, 255, 0);
+}
+
+button {
+  @include hassRegular;
+  padding: 0;
+}
+
+.wishList {
+  display: flex;
+  justify-content: center;
+  .wishWhole {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 130px;
+    .wishBox {
+      @include hassRegular;
+      .wishHeader {
+        @include flexBtwn;
+        width: 1080px;
```

```suggestion
  display: flex;
  justify-content: center;
  
  .wishWhole {
    display: flex;
    justify-content: space-around;
    margin-top: 130px;
    
    .wishBox {
      @include hassRegular;
      
      .wishHeader {
        @include flexBtwn;
        width: 1080px;
```
- 셀렉터 끼리는 한줄 띄워주세요
- 기본적인 컨벤션은 반드시 지켜주세요

---

수정했습니다